app/views/shared/_nav.html.erb

```diff
@@ -1,8 +1,10 @@
 <nav>
    <div class="nav-wrapper orange darken-3">
-     <a href="<%= root_path %>" class="brand-logo">Weasleys' Wizard Wheezes</a>
-     <ul id="nav-mobile" class="right hide-on-med-and-down">
-        <li><a href="/cart">Cart</a></li>
-      </ul>
-   </div>
+     <div class ="container"
```

Comment:
Looks good to me. I like the decision to put it after the nav-wrapper div, it looks a lot better that way.